al_aws_collector.js

```diff
@@ -175,14 +184,28 @@ class AlAwsCollector {
                         // when all else fails, stringify it the gross way with inspect
                         util.inspect(error);
             }
-            const status = this.prepareErrorStatus(errorString);
+            // post stream specific error
+            const status = streamType ? this.prepareErrorStatus(errorString, 'none', streamType) : this.prepareErrorStatus(errorString);
             this.sendStatus(status, () => {
                 context.fail(errorString);
             });
         } else {
             return context.succeed();
         }
     }
+    prepareHealthyStatus(collectionType, streamName = 'none') {
```

Comment:
one small nitpick here would be let's keep the ordering of the arguments consistent as well.